### DISCUSSION TITLE
Auto collapse function arguments from outline by default

### DISCUSSION THREAD

**Discussion by OlafvdSpek** [microsoft/pylance-release#2672] [Enhancement]
The 'outline' with function args takes up more space than the document itself..
IMO function args are of little value, could an option be added to exclude them? Maybe even exclude them by default.
Collapsing functions (manually) isn't a solution IMO.

https://github.com/microsoft/vscode/issues/133834

![image](https://user-images.githubusercontent.com/2218936/134957480-020dc9f2-f1c6-40e8-8853-baf594c20024.png)

**Comment by karthiknadig** [comment]
`JediLSP` does not provide function arguments in the outline. This is likely from `Pylance`.

/cc @jakebailey

**Comment by jakebailey** [comment]
Correct; we show all symbols. I'd transfer this so we can consider this behavior change.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
